developer/src/kmcmplib/tests/kmcompxtest.cpp

```diff
@@ -102,7 +104,7 @@ int main(int argc, char *argv[])
       std::istringstream(ErrNr) >> std::hex >> error_val;
 
       // check if error_val is in Array of Errors; if it is found return 0 (it's not an error)
-      for (int i = 0; i < error_vec.size() ; i++) {
+      for (std::vector<int>::size_type i = 0; i < error_vec.size() ; i++) {
```

Comment:
Would `size_t` work? That would be a bit simpler.
```suggestion
      for (size_t i = 0; i < error_vec.size() ; i++) {
```

Reply:
Seems to work. Done.